cmd/kured/main.go

```diff
@@ -91,6 +94,12 @@ func main() {
 		PreRun: flagCheck,
 		Run:    root}
 
+	rootCmd.PersistentFlags().BoolVar(&forceReboot, "force-reboot", false,
+		"force a reboot even if the drain is still running (default false)")
+	rootCmd.PersistentFlags().IntVar(&drainGracePeriod, "drain-grace-period", -1,
+		"grace period of time for pods to wait for the node drain in seconds (default -1)")
+	rootCmd.PersistentFlags().DurationVar(&drainTimeout, "drain-timeout", 0,
```

Comment:
In theory I'm fine with a zero drain timeout, as that is operationally equivalent to the current implementation (no timeout specified).

I think in the future it might be sensible to express an opinion here, as a node "stuck" in a drain operation can slow down kured-induced node reboots across the cluster.

Reply:
I agree with @jackfrancis . I would prefer set that in a different PR though, and in a different release (we have many things lined up for this one).

Reply:
Nice that you mention "infinite time", it's definitely good for our users!